src/services/machines/siopV2MachineService.ts

```diff
@@ -189,3 +189,45 @@ export const sendResponse = async (
 
   return response;
 };
+
+export const getFederationTrust = async (
+  context: Pick<SiopV2MachineContext, 'url' | 'authorizationRequestData' | 'trustAnchors'>,
+): Promise<Array<string>> => {
+  const {url, authorizationRequestData, trustAnchors} = context;
+
+  if (trustAnchors.length === 0) {
+    return Promise.reject(Error('No trust anchors found'));
+  }
+
+  if (!url) {
+    return Promise.reject(Error('Missing request url in context'));
+  }
+
+  if (!authorizationRequestData?.uri) {
+    return Promise.reject(Error('Missing authorization request data  uri in context'));
+  }
+
+  const parsedUrl = new URL(url);
+  const params = new URLSearchParams(parsedUrl.search);
+  const openidFederation = params.get('openid_federation');
```

Comment:
openid_federation is not listed as a valid query parameter in the OpenID4VP specification.

Reply:
yes, i think i will remove this, as it seems we have a spec for this and not some extension that was made specifically for this. this is coming from the vci solution